### PR TITLE
Added pandas built-in-function: get_dummies.

### PR DESCRIPTION
**Pull request by dkr2109**
### Description

Added pandas built-in-function: get_dummies.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - 'Variable Types'
+  - 'Variables'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'path/master-statistics-with-python'
+  - 'path/data-scientist'
+---
+
+The **.get_dummies()** function creates dummy (indicator) variables from categorical variables. Each level of the categorical variable gets its own column - a dummy variable. The dummy variables take on the value "1" if the original categorical variable was that level or "0" if not. 
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md:22]:
```suggestion
## Codebyte Example

```

Comment (caupolicandiaz) [reply content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md:22]: There should be a line break after the codebyte header, which is followed by a short description of the example.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@dkr2109 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+Title: '.get_dummies()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```
[the remark this answers is not in the document]
Comment (dkr2109) [reply content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md:2]: Done

File: content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - 'Statistics'
+  - 'Variable Types'
+  - 'Variables'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'path/master-statistics-with-python'
+  - 'path/data-scientist'
+---
+
+The **.get_dummies()** function creates dummy (indicator) variables from categorical variables. Each level of the categorical variable gets its own column - a dummy variable. The dummy variables take on the value "1" if the original categorical variable was that level or "0" if not. 
+
```
[the remark this answers is not in the document]
Comment (dkr2109) [reply content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md:19]: Added

File: content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md (hunk 1 above)
Comment (dkr2109) [reply content/pandas/concepts/built-in-functions/terms/get_dummies/get_dummies.md:20]: Done

**Comment by dkr2109** [comment]
@caupolicandiaz All comments are addressed. Please let me know what you think. Thanks!

File: content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+A number of other parameters are optional or have default arguments. Some of the key parameters are listed below.
+
+| Parameter Name | Data Type | Usage |
+| :--- | :--- | :--- |
+| `prefix` | str, list of str, or dict of str, default None | String to append to the beginning of DataFrame column names. If a list is passed, its length should be equal to the number of columns. A dictionary can also be passed; it should map column names to prefixes. |
+| `prefix_sep` | str, default '_' | If the `prefix` parameter is not `None`, this is a string appended after the prefix separating it from the level of the categorical variable. |
+| `dummy_na` | bool, default False | Adds a column to indicate NaNs. If false, NaNs are ignored. |
+| `columns` | list-like, default None | If a DataFrame is passed to the `data` parameter, a list of columns to can be passed to the `columns` parameter to be encoded as dummy variables. If `columns` is `None` then all the columns with object, string, or category dtype will be converted. |
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md:34]: I see a couple of parameters missing: sparse and dtype. Please add these as well

File: content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  - 'Data Science'
+  - 'Machine Learning'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Data'
+  - 'Encoding'
+  - 'Pandas'
+  - 'Scikit-learn'
+  - 'Statistics'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'path/data-scientist'
```
Comment (caupolicandiaz) [inline content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md:14]:
```suggestion
  - 'learn-python-3'
  - 'path/data-scientist'
```

File: content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+| `prefix_sep` | str, default '_' | If the `prefix` parameter is not `None`, this is a string appended after the prefix separating it from the level of the categorical variable. |
+| `dummy_na` | bool, default False | Adds a column to indicate NaNs. If false, NaNs are ignored. |
+| `columns` | list-like, default None | If a DataFrame is passed to the `data` parameter, a list of columns to can be passed to the `columns` parameter to be encoded as dummy variables. If `columns` is `None` then all the columns with object, string, or category dtype will be converted. |
+| `drop_first` | bool, default False | If `True`, this drops the first level of each variable to undergo dummy encoding. | 
+
+## Codebyte Example
+
+Creates a list of letters in the alphabet `abcs`, converts `abcs` to a Series, and prints the Series. Then uses the `.get_dummys()` function on the Series to create a DataFrame whose columns are indicator variables for each level of the single categorical variable in the Series. Finally, prints the DataFrame.
```
Comment (caupolicandiaz) [inline content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md:39]:
```suggestion
The code below creates a list based on the letters of the alphabet `abcs`, converts `abcs` to a Series, and prints the Series. Then the `.get_dummys()` function is applied to the Series to create a DataFrame whose columns are indicator variables for each level of the single categorical variable in the Series. Finally, the DataFrame is printed. 
```

Comment (caupolicandiaz) [reply content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md:39]: For future entries please try to develop original content for the examples. I think this is different enough from the pandas documentation to be included as is, but ideally we would prefer something unlike what's available on other sites.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@dkr2109 A few additional comments. Thanks

**Comment by dkr2109** [comment]
@caupolicandiaz I made all suggested edits. Please let me know what you think.

**Review by caupolicandiaz — APPROVED** [review]
@dkr2109 Looks good. I just made a few formatting changes to terms in the table so that this entry would be better aligned with our other pandas docs. This is ready for another review.

File: content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+| :------------- | :--------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| prefix         | str, list of str, or dict of str, default None | String to append to the beginning of DataFrame column names. If a list is passed, its length should be equal to the number of columns. A dictionary can also be passed, it should map column names to prefixes.                                    |
+| prefix_sep     | str, default '\_'                              | If the prefix parameter is not None, this is a string appended after the prefix separating it from the level of the categorical variable.                                                                                                          |
```
Comment (SSwiniarski) [inline content/pandas/concepts/built-in-functions/terms/get-dummies/get-dummies.md:33]:
```suggestion
| `prefix_sep`     | `str`, default '\_'                              | If the prefix parameter is not `None`, this is a string appended after the prefix separating it from the level of the categorical variable.                                                                                                          |
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dkr2109 I did a second review and left some comments, all having to do with formatting of terms.

**Comment by dkr2109** [comment]
@SSwiniarski Thanks for your review! I made all the suggested changes. Anything left to do?

**Review by SSwiniarski — APPROVED** [review]
@dkr2109 It looks good to me.

**Comment by SSwiniarski** [comment]
@dkr2109 Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/pandas/built-in-functions/get-dummies

image

**Pull request merged**